docs/blog/2018-10-16-unstructured-data/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Using unstructured data in Gatsby
+date: 2018-10-16
+author: Amberley Romo
+tags: ["sourcing", "data"]
+---
+
+When creating experiences on the web, one of the questions you can't get around asking (unless you're writing hard-coded HTML) is, "how do I get my data from point A (wherever it lives) to point B (where I want it)?". When wanting to take advantage of modern tooling like React, webpack, babel, etc., this can end up being a deceptively complex question.
```

Comment:
This initial paragraph could be streamlined:

"When creating web experiences, an inevitable question is, "how do I get my data from point A (the source) to point B (the component)?". This can end up being a deceptively complex question."

I'm not sure what modern tooling has to do with this question either so can be eliminated?